python/pyspark/broadcast.py

```diff
@@ -134,7 +137,15 @@ def value(self):
         """ Return the broadcasted value
         """
         if not hasattr(self, "_value") and self._path is not None:
-            self._value = self.load_from_path(self._path)
+            # we only need to decrypt it here when encryption is enabled and
```

Comment:
I think you can get in here when encryption off on executors and self._sc would be not definied

Reply:
sorry yeah i thought hasattr would take of that my bad earlier had the check self._sc is not None, will add it back in load_from_path